loader/convert_data.go

```diff
@@ -236,10 +251,25 @@ func parseTable(r *router.Table, schema, table, file string) (*tableInfo, error)
 		return nil, errors.Errorf("statement %s for %s/%s is not create table statement", statement, schema, table)
 	}
 
-	columns := make([]string, 0, len(ct.Cols))
+	var (
+		columns          = make([]string, 0, len(ct.Cols))
+		hasGeneragedCols = false
+		columnNameFields = ""
+	)
 	for _, col := range ct.Cols {
+		if len(col.Options) > 0 && col.Options[0].Tp == ast.ColumnOptionGenerated {
```

Comment:
`ColumnOptionGenerated` is always the first option in `col.Options`?

Reply:
changed to iterate all column options, which is more robust